README.md

```diff
@@ -37,27 +59,16 @@ tuple in a list (`[{}]`) to represent empty objects. jsn is incompatible with
 this format. While the getter (`jsn:get/2,3`) functions are generally functional;
 most other library functions are not, and may result in unpredictable behaviors.
 
-### Deprecated: encoding and decoding
+### Edoc generation broken by map support
 
-jsn will no longer be supporting encoding and decoding. See [below](#encode-decode)
-for more information.
+the `edoc` make target (and using `rebar3 edoc`) are currently broken due to a
+parser problem triggered by the `IF_MAPS(...)` macro used to implement the
+`map` format in a backwards-compatible fashion. Edoc support will be restored
+when this problem is addressed in Erlang or the `map` backwards-compatibility
+constructions are removed from jsn in a future version.
```

Comment:
It wouldn't hurt, although if they patch it in 19/20 it still won't help this project while we support older Erlangs.